packages/ra-core/src/auth/useLogoutIfAccessDenied.spec.tsx

```diff
@@ -142,76 +150,118 @@ describe('useLogoutIfAccessDenied', () => {
                     index++; // answers immediately first, then after 100ms the second time
                 }),
         };
-        const { queryByText } = renderInRouter(
-            <AuthContext.Provider value={delayedAuthProvider}>
-                <TestComponent />
-                <TestComponent />
-            </AuthContext.Provider>
+        render(
+            <MemoryRouter>
+                <AuthContext.Provider value={delayedAuthProvider}>
+                    <Routes>
+                        <Route
+                            path="/"
+                            element={
+                                <>
+                                    <TestComponent />
```

Comment:
There were 2 forbidden component in the test you replaced

Reply:
Fixed